### ISSUE TITLE
[Bug Report] Preview sometimes doesn't play on the first page

### ISSUE THREAD

**Issue by mrbrdo** [stashapp/stash#1257]
**Describe the bug**
Happened to me quite often that none of the mouseover previews on the page will play. Changing to another results page fixed the issue (both on the next page and if then going back they work on the first page too). This always happened to me on the first page, but not sure if that wasn't just by chance. Refreshing the page doesn't help, but going to another page in the results and then back does fix it. I am using the normal previous, not the image animated webP previews.

**Stash Version: (from Settings -> About):** all recent versions

**Desktop (please complete the following information):**
 - OS: OSX Big Sur
 - Browser Chrome
 - Version 89.0.4389.90

**Comment by InfiniteTF** [comment]
This is likely because Chrome requires user interaction before playing a video. Clicking anywhere on the page before hovering should consistently activate hover previews.

There are alternative ways to circumvent this by using autoplay, but it it still only works when audio is muted.

**Comment by mrbrdo** [comment]
I will try that the next time I encounter the issue (it's sporadic). So you're saying this issue wouldn't happen if the previews didn't play audio?

**Comment by InfiniteTF** [comment]
> I will try that the next time I encounter the issue (it's sporadic). So you're saying this issue wouldn't happen if the previews didn't play audio?

No, it'll happen regardless since we don't use autoplay, we explicitly play the video which I believe is restricted. Pornhub doesn't have this problem, presumably due to using autoplay instead.

**Comment by mrbrdo** [comment]
I see. What is the functional difference, why do you prefer to play it explicitly? As far as I remember it seems to work very similar to how it works on Pornhub.
Just out of interest.

**Comment by feederbox826** [comment]
> There are alternative ways to circumvent this by using autoplay, but it it still only works when audio is muted.


This is a restriction on the browser side, if you want to bypass this behaviour most browsers let you override autoplay from default to ON
